spark/src/main/java/org/opensearch/sql/spark/flint/FlintIndexMetadataReader.java

```diff
@@ -11,5 +11,5 @@ public interface FlintIndexMetadataReader {
    * @param indexDetails indexDetails.
    * @return jobId.
    */
-  String getJobIdFromFlintIndexMetadata(IndexDetails indexDetails);
+  FlintIndexMetadata getJobIdFromFlintIndexMetadata(IndexDetails indexDetails);
```

Comment:
Nit: change method name.
Also, return type in comments.

Reply:
done.